components/Toggle/Togglev2.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/_index';
+
+.root {
+	width: fit-content;
+	border: 0px !important;
+	font-weight: 500;
+	padding: 0.3125rem;
+	background: transparent !important;
+
+	.toggle {
+		@include flex(row, flex-start);
+		position: relative;
+		gap: 0.3375rem;
+		height: 100%;
+		background: transparent;
+
+		> button[data-elem='base-cell'].toggle-button {
+			position: relative;
+			z-index: 2;
+			background: transparent;
+			min-width: 5rem;
+			width: auto;
+			height: auto;
```

Comment:
Put the `border-radius` here, instead of duplicating the same in each sub level classes.